hamcrest/Hamcrest/File/IsExistingDirectory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @author Vasek Brychta <vaclav@brychtovi.cz>
+ */
+
+namespace Hamcrest\File;
+
+class IsExistingDirectory extends FileMatcher
+{
+	public function __construct()
+	{
+		parent::__construct('an existing directory', 'is not a directory');
+	}
+
+	protected function matchesFile(\SplFileInfo $file): bool
```

Comment:
Non a PHP 5.3+ compatible code. Likely you have more places, like this.

P.S.
I wonder any none of the builds have failed. Ha-ha, they don't include PHP 5.x in even though it's supported according to `composer.json`.

I guess that issue needs to be fixed in a separate PR if you're up for it.